python/pyspark/ml/param/_shared_params_code_gen.py

```diff
@@ -51,12 +51,12 @@ def _gen_param_header(name, doc, defaultValueStr):
     """
 
     # a placeholder to make it appear in the generated doc
-    $name = Param(Params._dummy(), "$name", "$doc")
+    $name = Param(Params._dummy(), "$name", "$doc.")
```

Comment:
I did it the other way since it would be less changes, but yah that sounds like this would be more readable. I'll swap it :)